phpBB/includes/functions_acp.php

```diff
@@ -80,10 +80,10 @@ function adm_page_header($page_title)
 		'U_INDEX'				=> append_sid("{$phpbb_root_path}index.$phpEx"),
 
 		'T_IMAGES_PATH'			=> "{$phpbb_root_path}images/",
-		'T_SMILIES_PATH'		=> "{$phpbb_root_path}{$config['smilies_path']}/",
+		'T_SMILIES_PATH'		=> "{$phpbb_root_path}",	// {$config['icons_path']}/ is not used as for 3.2 smiley path is dumped into the DB for every smiley
 		'T_AVATAR_PATH'			=> "{$phpbb_root_path}{$config['avatar_path']}/",
 		'T_AVATAR_GALLERY_PATH'	=> "{$phpbb_root_path}{$config['avatar_gallery_path']}/",
-		'T_ICONS_PATH'			=> "{$phpbb_root_path}{$config['icons_path']}/",
+		'T_ICONS_PATH'			=> "{$phpbb_root_path}",	// {$config['icons_path']}/ is not used as for 3.2 icon path is dumped into the DB for every icon
```

Comment:
Really, I would just write `$phpbb_root_path`. (or `$web_path` in functions.php, see above)